app/models/address.rb

```diff
@@ -11,7 +11,13 @@
 #  street       :string
 #  created_at   :datetime         not null
 #  updated_at   :datetime         not null
+#  denounce_id  :bigint
+#
+# Indexes
+#
+#  index_addresses_on_denounce_id  (denounce_id)
 #
```

Comment:
If all these comments are not necessary you can remove them, so that the code looks cleaner.